base/src/main/java/com/smartdevicelink/managers/screen/menu/BaseMenuManager.java

```diff
@@ -175,7 +175,11 @@ public void setMenuCells(@NonNull List<MenuCell> cells){
 		if (currentHMILevel == null || currentHMILevel.equals(HMILevel.HMI_NONE) || currentSystemContext.equals(SystemContext.SYSCTXT_MENU)){
 			// We are in NONE or the menu is in use, bail out of here
 			waitingOnHMIUpdate = true;
-			waitingUpdateMenuCells = new ArrayList<>(clonedCells);
+			if (clonedCells == null) {
```

Comment:
Couldn't this be simplified like:
```
             waitingUpdateMenuCells = new ArrayList<>();
            if(clonedCells != null ){
                waitingUpdateMenuCells.addAll(clonedCells);
            }
```

Reply:
Yes, that looks cleaner. I will update the PR.